src/test/java/org/sagebionetworks/bridge/services/ParticipantFileServiceTest.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.services;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.GeneratePresignedUrlRequest;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.sagebionetworks.bridge.TestConstants;
+import org.sagebionetworks.bridge.config.BridgeConfig;
+import org.sagebionetworks.bridge.dao.ParticipantFileDao;
+import org.sagebionetworks.bridge.exceptions.BadRequestException;
+import org.sagebionetworks.bridge.exceptions.EntityAlreadyExistsException;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.models.files.ParticipantFile;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.net.URL;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+public class ParticipantFileServiceTest {
+
+    private static final String UPLOAD_BUCKET = "file-bucket";
+
+    @Mock
+    ParticipantFileDao mockFileDao;
+
+    @Mock
+    BridgeConfig mockConfig;
+
+    @Mock
+    AmazonS3 mockS3Client;
+
+    @InjectMocks
+    ParticipantFileService service;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        MockitoAnnotations.initMocks(this);
+
+        when(mockConfig.get("participant-file.bucket")).thenReturn(UPLOAD_BUCKET);
+        service.setConfig(mockConfig);
+
+        when(mockS3Client.generatePresignedUrl(any())).thenAnswer(i -> {
+            GeneratePresignedUrlRequest request = i.getArgument(0);
+            String filePath = request.getKey();
+            return new URL("https://" + UPLOAD_BUCKET + "/" + filePath);
+        });
+    }
+
+    @Test
+    public void getParticipantFiles() {
+        service.getParticipantFiles("test_user", "dummy-key", 20);
+
+        verify(mockFileDao).getParticipantFiles("test_user", "dummy-key", 20);
+    }
+
+    @Test
+    public void getParticipantFilesNoOffsetKey() {
+        service.getParticipantFiles("test_user", null, 20);
+
+        verify(mockFileDao).getParticipantFiles("test_user", null, 20);
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void getParticipantFilesPageSizeTooSmall() {
+        service.getParticipantFiles("test_user", null, 1);
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void getParticipantFilesPageSizeTooLarge() {
+        service.getParticipantFiles("test_user", null, 5000);
+    }
+
+    @Test
+    public void getParticipantFile() {
```

Comment:
This test should verify arguments into generatePresignedUrl().

Same with createParticipantFile().